packages/storefront-lwc/src/modules/commerce/data/forms.js

```diff
@@ -0,0 +1,78 @@
+/*
+    Copyright (c) 2020, salesforce.com, inc.
+    All rights reserved.
+    SPDX-License-Identifier: BSD-3-Clause
+    For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+*/
+/**
+ * Form Helper
+ * @type {{validateForm: FormHelper.validateForm, validate: FormHelper.validate}}
+ */
+export const FormHelper = {
+    validateForm: formData => {
+        let isValid = true;
+        Object.keys(formData).forEach(name => {
+            isValid = FormHelper.validate(formData, name) && isValid;
+        });
+
+        return isValid;
+    },
+
+    validate: (formData, name, control) => {
+        control =
+            control ||
+            document.querySelector(
```

Comment:
Should we be using `document.querySelector`?

Reply:
File deleted